src/components/box/Box.jsx

```diff
@@ -68,64 +77,116 @@ type BoxPropsType = {
    * @default false
    */
   full?: ?boolean,
   /**
    * Specify padding sizes for inner content
-   * @see https://styleguide.brainly.com/latest/docs/containers.html#box
+   * @example <Box padding="large-padding">
+   *            some text
+   *          </Box>
+   * @see no-padding https://styleguide.brainly.com/latest/docs/interactive.html?padding=no-padding#boxes
+   * @see small-padding https://styleguide.brainly.com/latest/docs/interactive.html?padding=small-padding#boxes
+   * @see xsmall-padding https://styleguide.brainly.com/latest/docs/interactive.html?padding=xsmall-padding#boxes
+   * @see xxsmall-padding https://styleguide.brainly.com/latest/docs/interactive.html?padding=xxsmall-padding#boxes
+   * @see large-padding https://styleguide.brainly.com/latest/docs/interactive.html?padding=large-padding#boxes
    */
   padding?: ?PaddingType,
   /**
    * Specify src for image, which will be displayed INSTEAD of inner content
-   * @see https://styleguide.brainly.com/latest/docs/containers.html#box
+   * @example <Box imgSrc="https://some_url">
+   *            THIS CONTENT WILL
+   *            BE REPLACED BY IMAGE
+   *          </Box>
    */
   imgSrc?: ?string,
   /**
    * Specify if border-shadow should be displayed around the **<Box/>**
-   * @see https://styleguide.brainly.com/latest/docs/containers.html#box
+   * @example <Box shadow>
+   *            some text
+   *          </Box>
+   * @see shadow=true https://styleguide.brainly.com/latest/docs/interactive.html?shadow=true#boxes
+   * @see shadow=false https://styleguide.brainly.com/latest/docs/interactive.html?shadow=false#boxes
    * @default false
    */
   shadow?: ?boolean,
   /**
    * Specify if **<Box/>** should have flat corners
-   * @see https://styleguide.brainly.com/latest/docs/containers.html#box
+   * @example <Box noBorderRadius>
+   *            some text
+   *          </Box>
+   * @see noBorderRadius=true https://styleguide.brainly.com/latest/docs/interactive.html?noBorderRadius=true#boxes
+   * @see noBorderRadius=false https://styleguide.brainly.com/latest/docs/interactive.html?noBorderRadius=true#boxes
    * @default false
    */
   noBorderRadius?: ?boolean,
   /**
    * Color of **close** icon. Prop used only in case if **onClose** prop specified as well
-   * @example
-   * // right
-   * <Box onClose={() => doSomething()} closeIconColor="DARK">some text</Box>
-   * // wrong
-   * <Box closeIconColor="DARK">some text</Box>
-   * @see https://styleguide.brainly.com/latest/docs/containers.html#box
+   * @example // right
+   *          <Box
+   *            onClose={() => doSomething()}
+   *            closeIconColor="DARK"
+   *          >
+   *            some text
+   *          </Box>
+   *
+   *          // wrong
+   *          <Box closeIconColor="DARK">
+   *            some text
+   *          </Box>
+   * @todo interactive example for this prop missing. Please check twig-compatible docs to see more
```

Comment:
I would say: todo add interactive example for this prop.

Reply:
done